src/main/java/seedu/eylah/Eylah.java

```diff
@@ -1,37 +1,86 @@
 package seedu.eylah;
 
+import java.util.Scanner;
 import java.util.logging.Logger;
 
-import seedu.eylah.addressbook.logic.Logic;
-import seedu.eylah.addressbook.model.Model;
 import seedu.eylah.commons.core.LogsCenter;
 import seedu.eylah.commons.core.Version;
+import seedu.eylah.diettracker.logic.Logic;
+import seedu.eylah.diettracker.logic.LogicManager;
+import seedu.eylah.diettracker.logic.commands.CommandResult;
+import seedu.eylah.diettracker.logic.commands.exceptions.CommandException;
+import seedu.eylah.diettracker.logic.parser.exceptions.ParseException;
+import seedu.eylah.diettracker.model.Model;
+import seedu.eylah.diettracker.model.ModelManager;
 
+/**
+ * The main entry for the EYLAH.
+ */
 public class Eylah {
 
     public static final Version VERSION = new Version(0, 1, 0, true);
 
     private static final Logger logger = LogsCenter.getLogger(Eylah.class);
 
-    protected Logic logic;
-    protected Model model;
+    protected Logic dietLogic;
+    protected Model dietModel;
+    protected seedu.eylah.expensesplitter.logic.Logic splitterLogic;
+    protected seedu.eylah.expensesplitter.model.Model splitterModel;
```

Comment:
I understand there there is no aliasing supported by Java. Would it prudent to name the Logic and Model classes of each more specifically? Something to consider in future. Otherwise, LGTM.

Reply:
Yap, will definitely work on that during v1.3. For now, I just create a simple interface for the developer to test their command and function.